config/default.js

```diff
@@ -71,6 +71,9 @@ module.exports = {
       showWarning: true,
       checkboxText: 'By checking here you confirm that the information is correct',
     },
+    seldon: {
+      versions: ['v1', 'v2'],
```

Comment:
These should point to the Docker Seldon versions we use in the application.  They should have both an 'image' and a 'name', as the image may not be user readable on some registries.